src/runtime_src/core/tools/xbmgmt2/ReportPlatform.cpp

```diff
@@ -255,6 +255,7 @@ ReportPlatform::getPropertyTree20202( const xrt_core::device * device,
     pt_status.put("shell", same_shell( pt_current_shell.get<std::string>("vbnv", ""), 
               pt_current_shell.get<std::string>("id", ""), installedDSA));
     pt_status.put("sc", same_sc( pt_current_shell.get<std::string>("sc_version", ""), installedDSA));
+    pt_status.put("is_mfg", xrt_core::device_query<xrt_core::query::is_mfg>(device));

```

Comment:
let's rename "is_mfg" to "is_factory" and store this value as a true/false instead of 0/1 if that's not already the case

Reply:
Its already a bool from the query request, but the name change is good :)